stacks/serverless/dovetail-cdn-arranger.yml

```diff
@@ -0,0 +1,133 @@
+# stacks/serverless/dovetail-cdn-arranger.yml
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Dovetail CDN Arranger lambda function
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+Parameters:
+  CodeS3Bucket:
+    Type: String
+  CodeS3ObjectVersion:
+    Type: String
+  EnvironmentType:
+    Type: String
+  DovetailHost:
+    Type: AWS::SSM::Parameter::Value<String>
+  DovetailToken:
+    Type: AWS::SSM::Parameter::Value<String>
+  DynamodbTableArn:
+    Type: AWS::SSM::Parameter::Value<String>
+Resources:
+  DovetailCdnArrangerWorkspaceBucket:
+    Type: AWS::S3::Bucket
```

Comment:
This should have a `DeletionPolicy` and `UpdateReplacePolicy` set to `Retain` even though we don't actually want to retain the buckets or data. Because it's unlikely this bucket would exist and be empty, not having the policies would mess with tearing down a stack.